File: ascii_to_binary.js
function asciiToBinary(ascii) {
  var binary = "";
  for (var i = 0; i < ascii.length; i++) {
    binary += String(ascii.charCodeAt(i)).padStart(8, '0');
  }
  return binary;
}

// Example
var ascii = "Hello, World!";
var binary = asciiToBinary(ascii);
console.log("The binary representation of " + ascii + " is " + binary);

File: ascii_to_binary.py
def ascii_to_binary(ascii_text):
  binary_text = ""
  for char in ascii_text:
    binary_text += format(ord(char), "08b")
  return binary_text

if __name__ == "__main__":
  ascii_text = input("Enter ASCII text: ")
  binary_text = ascii_to_binary(ascii_text)
  print("The binary representation of {} is {}".format(ascii_text, 
binary_text))
